scripts/pylint_extensions.py

```diff
@@ -18,6 +18,7 @@
 presubmit checks.
 """
 
+import re
```

Comment:
Alphabetize the imports.

Reply:
I tried to... but it the pre-commit checks are not allowing me to do so and throw incorrect import order error.

Reply:
Checked this out, this ordering is correct. Thanks!